cmd/tether/attach.go

```diff
@@ -289,7 +293,9 @@ func backchannel(ctx context.Context, conn net.Conn) error {
 	go func() {
 		select {
 		case <-ctx.Done():
-			conn.Close()
+			if conn != nil {
```

Comment:
Oh, the line 274 is a leftover that I forgot to remove. And you are right about the nil check between 296 and 297, to be honest I'm not sure what I was thinking 10 days ago when I pasted that to the issue :), let me check!